### ISSUE TITLE
NTR: [positive regulation of ATPase activity, coupled]

### ISSUE THREAD

**Issue by robnash** [geneontology/go-ontology#23227]
Please provide as much information as you can: 

* **Suggested term label:**

positive regulation of ATPase activity, coupled

* **Definition (free text)**

Any process that activates or increases the frequency, rate or extent of ATPase activity, coupled.

* **Reference, in format PMID:#######**
* 
https://pubmed.ncbi.nlm.nih.gov/34471130/
https://pubmed.ncbi.nlm.nih.gov/35308047/

* **Gene product name and ID to be annotated to this term**

YTA7 (S. cerevisiae) aka YGR270W, SGD:S000003502, P40340

* **Parent term(s)**

GO:0042623  ATPase activity, coupled

* **Children terms (if applicable)** Should any existing terms that should be moved underneath this new proposed term?

none

* **Synonyms (please specify, EXACT, BROAD, NARROW or RELATED)**

NARROW 	activation of ATP phosphohydrolase (polypeptide-unfolding),
NARROW 	activation of chaperonin ATPase activity, 
NARROW 	activation of non-chaperonin molecular chaperone ATPase activity, 
NARROW 	activation of nucleoplasmin ATPase activity, 
NARROW 	activation of peroxisome-assembly ATPase activity, 
NARROW 	activation of vesicle-fusing ATPase activity; 
NARROW 	positive regulation of ATP phosphohydrolase (polypeptide-unfolding), 
NARROW 	positive regulation of chaperonin ATPase activity, 
NARROW 	positive regulation of non-chaperonin molecular chaperone ATPase activity, 
NARROW 	positive regulation of nucleoplasmin ATPase activity, 
NARROW 	positive regulation of peroxisome-assembly ATPase activity, 
NARROW 	positive regulation of vesicle-fusing ATPase activity 
* **Cross-references**
* For enzymes, please provide RHEA and/or EC numbers. 
* Can also provide MetaCyc, KEGG, Wikipedia, and other links.

* **Any other information**

**Comment by pgaudet** [comment]
@robnash Did you mean to close this? 

I dont know which version of the ontology you are looking at. GO:0042623 has been merged into GO:0016887 ATP hydrolysis activity - see https://github.com/geneontology/go-ontology/issues/21612

We dont create 'regulation of MF' in the BP aspect; we are trying to weed those out (but it takes some time). 

Please use 'GO:0001671 ATPase activator activity' instead. 

Thanks, Pascale

**Comment by robnash** [comment]
Hi,
Thanks for the feedback. Yes, I realized the merge after I submitted the annotation, and yes there was a duplicate which was why I closed this one. Since the regulator is a cyclin-dependent kinase complex I am not sure the 'GO:0001671 ATPase activator activity' works here. It is through phosphorylation that the activity of the ATPase is increased. Since the definition "Binds to and increases the activity of an ATP hydrolysis activity." suggests a stable binding and activation such as you would get with a chaperone/co-chaperone I may not be able to make the annotation.
